### PR TITLE
"fixed the blocker"

### LINKED ISSUE #802: The tab order should follow the mockup.
In the single result page, the tab on the right section should have the following order and names:
1. Information
2. Attribution
3. Share

### PR DESCRIPTION
**Pull request by samcaspus**
Fixes #802 

Fixed the tab order As well as the display is also fixed correctly 

The tab order is not only fixed but the display after you click on the tab is also restructured. 
Minor changes made there
---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: src/components/PhotoDetails.vue [hunk 1]
```diff
@@ -13,26 +13,26 @@
...
-          </li>
           <li :class="tabClass(1, 'a')">
             <a href="#panel1" :aria-selected="activeTab == 1" @click.prevent="setActiveTab(1)">
               Info
             </a>
           </li>
-          <li :class="tabClass(2, 'a')" v-if="watermarkEnabled">
-            <a href="#panel2" :aria-selected="activeTab == 2" @click.prevent="setActiveTab(2)">
-              Download
+          <li :class="tabClass(0, 'a')">
```
Comment (brenoferreira) [inline src/components/PhotoDetails.vue]: You should change the order of the markup elements, not the order of the indices

**Comment by samcaspus** [comment]
## Page Display

I believe the order and the tab is working as desired.

## Page with information tab
image

## Page with attribute tab
image

## Page with share tab
image

## Page with download tab
image

File: src/components/PhotoDetails.vue (hunk 1 above)
Comment (samcaspus) [reply src/components/PhotoDetails.vue]: @brenoferreira I believe I made the necessary changes please do review.

**Comment by brenoferreira** [comment]
Looks good 👍

**Pull request merged**